addons/l10n_dk_oioubl/data/oioubl_templates.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="oioubl_PaymentTermsType" inherit_id="account_edi_ubl_cii.ubl_20_PaymentTermsType">
```

Comment:
I think it needs `primary="True"`.

Otherwise, it is modifying in place `account_edi_ubl_cii.ubl_20_PaymentTermsType` which is by default assigned here on:
https://github.com/odoo/odoo/blob/1b504038ce270540ddab28db4ab995d22f2ab2a7/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py#L513

so it's the default template for all models inheriting (directly or indirectly) from ubl 2.0

Reply:
Yes, thank you!